projects/libyuv/project.yaml

```diff
@@ -4,10 +4,14 @@ primary_contact: "mikhal@webrtc.org"
 auto_ccs:
     - "fbarchard@google.com"
     - "frkoenig@google.com"
+    - "byone.heng@google.com"
 fuzzing_engines:
     - libfuzzer
     - afl
 sanitizers:
     - address
     - memory
     - undefined
+architectures:
+    - x86_64
```

Comment:
libyuv supports arm and mips as well.  Not sure if they are supported by fuzz

Reply:
https://google.github.io/oss-fuzz/getting-started/new-project-guide/
![Uploading image.png…]()